ioctl/cmd/action/actionhash.go

```diff
@@ -241,6 +242,11 @@ func printReceiptProto(receipt *iotextypes.Receipt) string {
 		result += fmt.Sprintf("\ncontractAddress: %s %s", receipt.ContractAddress,
 			Match(receipt.ContractAddress, "address"))
 	}
+	if len(receipt.Logs) > 0 {
```

Comment:
display bucket index in 'ioctl action hash' for staking action

Reply:
example:

status: 209 
actHash: 7aa5544b4fb188223bbcdf02b7ba4e7f83de17c50d3b2935959550979e871be9
blkHeight: 3310128
gasConsumed: 10000
logs: 1
contractAddress: io1qnpz47hx5q6r3w876axtrn6yz95d70cjl35r53 
Bucket index: 33